tools/test/test262/make_excludelist.py

```diff
@@ -86,13 +88,15 @@ def main():
                         help='path to the engine to be tested (default: %(default)s)')
     parser.add_argument('--arch', metavar='NAME', choices=['x86', 'x86_64'], default='x86_64',
                         help='architecture the engine was built for (%(choices)s; default: %(default)s)')
+    parser.add_argument('--excludelist', metavar='PATH', default=DEFAULT_EXCLUDE_LIST,
+                        help='path to the exclude list (default: %(default)s)')
```

Comment:
Why did you add this argument? Is this argument really used?
Currently, it looks that the fixed file (DEFAULT_EXCLUDE_LIST) is used only.
`args.excludelist` is nowhere used.

Reply:
In line `131` and `132` It's used, but on better thought i think you're right it's not necessary, so i will delete it.